src/components/widgets/accounts-comparison/accounts-comparison.directive.coffee

```diff
@@ -22,43 +22,29 @@ module.controller('WidgetAccountsComparisonCtrl', ($scope, $q, ChartFormatterSvc
   # --------------------------------------
   w.initContext = ->
     $scope.isDataFound = w.content? && !_.isEmpty(w.content.complete_list)
-    # parameters for params-checkboxes.directive
+    # defines the available options for params-checkboxes.directive
     $scope.comparisonModeOptions = [{
       id: 'compare_accounts',
       label: 'Compare matching accounts across your companies',
       value: false,
       onChangeCallback: $scope.multiCompanyComparisonOnChange
```

Comment:
@xaun: can be changed to "$scope.updateSettings()" instead of defining a new function